DELIVERY.README.txt

```diff
@@ -63,10 +63,8 @@ Contains two type of reports, snpEff summary and sample report.
    |--SampleN
       |--00-Report
          |--SampleID_ign_sample_report.html
-         |--SampleID.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf.gz.snpEff.summary.csv
-         |--SampleID.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf.gz.snpEff.summary.genes.txt
-         |--SampleID.clean.dedup.recal.bam.recalibrated.snp.annotated.vcf.gz.snpEff.summary.csv
-         |--SampleID.clean.dedup.recal.bam.recalibrated.snp.annotated.vcf.gz.snpEff.summary.genes.txt
+         |--SampleID.clean.dedup.recal.bam.raw.annotated.vcf.gz.snpEff.summary.csv
+         |--SampleID.clean.dedup.recal.bam.raw.annotated.vcf.gz.snpEff.summary.genes.txt
```

Comment:
Is this the actual files that will be delivered? The corresponding expression in the taca config is `- <ANALYSISPATH>/piper_ngi/07_variant_calls/<SAMPLEID>.*.snpEff.summary.*`